uwblueprint/paramedics-web
# add delete patient status

## Overview
>[Code review doc for reference](https://www.notion.so/uwblueprintexecs/Code-Review-a21fc85b00394f488e92d9d605f6b2bc)

add delete patient status

[Notion task](https://www.notion.so/uwblueprintexecs/add-delete-patient-statuses-b3737d7a302c45a19446f6758f608325)


## Changes
- add delete patient status
  - no existing entries in your db should be affected

## Testing
- if you haven't already, pull latest `paramedics-builder` changes and build new `builder` binary
- rebuild your backend container first (so the new node module is properly installed)
  - this will run the new migration (as migrations run on every container start)
- shell into your db container - from your local paramedics-web, `../builder -e db`
  - run `\dT+ enum_patients_status`
  - check that the `DELETED` enum is there
![image](https://user-images.githubusercontent.com/59516223/86542226-c4639100-bee1-11ea-849a-7364c790db9c.png)
---
#### OPTIONAL: run the down migration and check that the `DELETED` enum is gone
- shell into your backend container - from your local paramedics-web, `../builder -e shell`
  - run `sequelize db:migrate:undo`
  - check down migration runs correctly
- leave backend container and shell into your db container - from your local paramedics-web, `../builder -e db`
  - run `\dT+ enum_patients_status`
  - check that the `DELETED` enum is gone
- go back to backend and run `sequelize db:migrate` just to be safe afterwards

## Screenshots
